warehouse/ingest-core/src/main/java/datawave/iterators/ZeroUIDCountMappingIterator.java

```diff
@@ -0,0 +1,40 @@
+package datawave.iterators;
+
+import org.apache.accumulo.core.data.Key;
+import org.apache.accumulo.core.data.Value;
+import org.apache.accumulo.core.iterators.IteratorEnvironment;
+import org.apache.accumulo.core.iterators.SortedKeyValueIterator;
+import org.apache.accumulo.core.iterators.WrappingIterator;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+
+import datawave.ingest.protobuf.Uid;
+
+public class ZeroUIDCountMappingIterator extends WrappingIterator implements SortedKeyValueIterator<Key,Value> {
```

Comment:
This class needs to be removed now that the zero count case is handled elsewhere

Reply:
done